crates/ollama/src/ollama.rs

```diff
@@ -148,6 +157,11 @@ impl ChatRequest {
         self.tools = tools;
         self
     }
+
+    pub fn with_images(mut self, images: Vec<String>) -> Self {
```

Comment:
Is this actually used anywhere? If not remove it

Reply:
Removed this.